packages/integrations/cloudflare/test/fixtures/vite-plugin/astro.config.mjs

```diff
@@ -11,6 +11,9 @@ export default defineConfig({
 		imageService: 'cloudflare-binding',
 		sessionKVBindingName: "SESSION"
 	}),
+	build: {
+		inlineStylesheets: 'never'
```

Comment:
Was this changed for testing something in particular?

Reply:
Yeah, to make sure the CSS was being built.